commands/signature.py

```diff
@@ -0,0 +1,82 @@
+import sublime_plugin
+
+from ..libs import *
+from ..libs.viewhelpers import *
+from ..libs.reference import *
+
+
+class TypescriptSignaturePanel(sublime_plugin.TextCommand):
+    def is_enabled(self):
+        return is_typescript(self.view)
+
+    def run(self, text):
+        print('TypeScript signature panel triggered')
```

Comment:
It is a little ambiguous right now, my understanding is for messages that we definitely will output we can use either ``logger.log.info`` or ``logger.log.warn`` or ``print``; for messages only used for debugging purpose we should use ``logger.log.debug``, that way we can turn off those messages by setting the global log level higher than ``DEBUG`` which still keep the code.